feg/radius/src/modules/analytics/xwf_vendors.go

```diff
@@ -20,7 +20,6 @@ type Vendor int
 // Enum list of vendors supported by Express-Wifi program in its wider meaning (Wifi, Carrier Wifi, Standalone, ...).
 const (
```

Comment:
Do we need this file at all at this point?

Reply:
No, we don't. Good catch!